hw/ip/otbn/doc/_index.md

```diff
@@ -607,8 +622,8 @@ Before being stored in SRAM, the data word with the attached Integrity Protectio
 The scrambling is reversed on a read.
 
 The ephemeral memory scrambling key and the nonce are provided by the [OTP block]({{<relref "/hw/ip/otp_ctrl/doc" >}}).
-They are set once when OTBN block is reset, and changed whenever a secure wipe of the data memory is performed.
-(TODO: Link to it once we have specified secure wipe.)
+They are set once when OTBN block is reset, and changed whenever a [secure wipe]({{<relref "#design-details-secure-wipe">}}) of the data memory is performed.
```

Comment:
This link is to the wrong section: it should point at the DMEM secure wipe, I think.